app/controllers/schools_controller.rb

```diff
@@ -23,13 +17,13 @@ def edit
   # POST /schools
   def create
     School.create(school_params)
```

Comment:
Something like this might work to remove that hidden field from the school new view. I'm not sure I have this 100% correct, but hopefully this gets the idea across.

```suggestion
    School.create(school_params.merge(admin: current_user))
```